### PR TITLE
Restructing with es6

### PR DESCRIPTION
**Pull request by Owusu-Desmond**
## Awesome Book refracturing with ES6

In this pull request, I update JavaScript file by using ES6 syntax.

## Changes Made
- [ ] Create time and date container in HTML to display time and date
- [ ] I grouped Classes and functions into modules
- [ ] I used es6 syntax that is I changed real functions to arrow functions
- [ ] I styled time and date container
- [ ] Fixing all Linter Errors

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Owusu-Desmond,** 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## **Invalid Code Review Request**

You have submitted a project, that _**your project failed to check all linter ☑️**_

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// import modules
+import Book from './modules/Book.js';
+import Storage from './modules/Storage.js';
+import UI from './modules/UI.js';
```
Comment (SajjadAhmad14) [inline index.js:4]: PR linter checks are still not green and failing. So kindly fix the errors before asking for another review.

App navigation is confusing for a user. User when clicks on `Contact` link the form is open to add book. Kindly see the screenshots of navigation bar to understand the issue and resolve it.
contact
add


lists

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Desmond, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by topeogunleye — APPROVED** [review]
### Hi @Owusu-Desmond, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**